guides/frontend-accelerators.md

```diff
@@ -0,0 +1,61 @@
+# Frontend accelerators
+
+## Getting started with Angular 
+
+{% hint style="info" %}
+You need to have Node.js and NPM installed. Follow [this guide](prepare-your-environment.md#install-node-js-and-npm) for installing them.
+{% endhint %}
+
+### test 
+
+```bash
+npm install -g @angular/cli
+```
+
+You can verify that the Angular CLI has been installed correctly by checking its version using: `ng -v`.
+
+Create a workspace with an application project using the Angular CLI command:
+
+```bash
+ng new <appname>
+```
+
+When promopted, select Yes for Angular routing. And preferably SCSS, SASS or LESS for the stylesheet format.
+
+The CLI will generate the basic skeleton for your application, and install the packages necessary for the application to run.
+
+Switch to the generated directoy `cd <appname>`. 
+
+Run `ng serve` to compile the project and start a web server. By default the web server is launched on port 4200. 
+
+You can access the project by typing http://localhost:4200/ in a browser. You should be able to see a splash screen with the Angular logo on it. 
+
+For the frontend to work with the APIs, you need to configure the environment files found in at the project directory `/src/environments/`. Refer to [Configuring Frontend](https://docs.cxcloud.com/setting-up-a-cxcloud-project/generating-a-frontend#configuration) for more information. 
+
+
+### Tools recommendations
+
+Install [Prettier](https://prettier.io/docs/en/install.html) using: `npm install --save-dev --save-exact prettier`
```

Comment:
For prettier and husky we would need to mention specific configuration which should be done in package.json. I can help with it, just recently did configuration for latest prettier and husky